app/controllers/browse_everything/authorizations_controller.rb

```diff
@@ -22,6 +26,28 @@ def show
       head(:not_found)
     end
 
+    def update
+      authorizations = Authorization.find_by(uuid: id)
+      raise ResourceNotFound if authorizations.empty?
+
+      authorization = authorizations.first
+      @authorization = authorization.update(**authorization_attributes)
+      @authorization.save
+
+      @serializer = AuthorizationSerializer.new(@authorization)
+      serialized_json = @serializer.serialized_json
+      serialized = JSON.parse(serialized_json)
+      # I do not know why this is happening
```

Comment:
This seems to be an issue with the `fast_jsonapi` serialization for `Authorizations`